linux_os/guide/services/cron_and_at/service_atd_disabled/tests/service_disabled.pass.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 #
-# profiles = xccdf_org.ssgproject.content_profile_standard
+# packages = at
```

Comment:
Should the `yum` command be removed, then?

Reply:
Not yet, there is no support for package metadata, but it is coming.